graphix/gflow.py

```diff
@@ -372,34 +303,36 @@ def search_neighbor(node, edges):
     return N
 
 
-def find_flow(g, v_in, v_out, meas_plane=None, timeout=100):
+def find_flow(graph, input, output, meas_planes=None, mode="single"):
```

Comment:
what's the mode argument?

Reply:
I missed the explanation, but it is the same option of `gflow` function.

Reply:
please explain what it is in the docstring.